cloudwatch/cloudwatch.go

```diff
@@ -461,8 +464,9 @@ func (output *OutputPlugin) putLogEvents(stream *logStream) error {
 			return err
 		}
 	}
+	output.processRejectedEventsInfo(response)
```

Comment:
huh were we need calling this before?

Reply:
I wasn't... I wrote the function and then forgot to use it... I'll break it out into a separate commit